### PR TITLE
Update AUTHORS and .mailmap

### PR DESCRIPTION
**Pull request by thaJeztah**
ping @AkihiroSuda @vdemeester PTAL

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1690,6 +1700,7 @@ Sergio Lopez <slp@redhat.com>
 Serhat Gülçiçek <serhat25@gmail.com>
 SeungUkLee <lsy931106@gmail.com>
 Sevki Hasirci <s@sevki.org>
+sh7dm <d3dx12.xx@gmail.com>
```
Comment (thaJeztah) [inline AUTHORS]: oh, hold on; one more to fix

Comment (thaJeztah) [reply AUTHORS]: done

**Review by AkihiroSuda — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]
LGTM :tiger:

**Comment by thaJeztah** [comment]
merging; Linux CI is broken due to the Microsoft images being removed from Docker Hub

**Pull request merged**